.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: 🐜 Bug Report
+about: Create a report to help us improve
+title: '[bug] '
+labels: bug
+---
+
+<!-- Please make sure you are posting an technical issue related to Gridcoin. --> 
+
+<!-- For general questions about Gridcoin or wallet recovery please use one of the various communities:
+* [Gridcoin on reddit](https://www.reddit.com/r/gridcoin/)
+* [Discord](https://discord.gg/jf9XX4a) -->
+
+<!-- ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION. -->
+
+# Bug Report
+
+**Current behavior:**
+<!-- Describe how the bug manifests. -->
+
+**Expected behavior**
+<!-- Describe what the behavior would be without the bug. -->
+
+**Steps to reproduce:**
+<!--  Please explain the steps required to duplicate the issue, especially if you are able to provide a sample application or sample code -->
+
+**Screenshots**
+<!-- If the issue is related to the GUI, screenshots can be added to this issue via drag & drop. -->
+
+**Gridcoin version**
+<!-- List the version number/commit ID, and if it is an official binary, self compiled or a distribution package such as PPA. 
+If you are not on the latest Leisure release please try updating to that and see if your issue still persists. -->
+
+**Machine specs**
+- OS:
+- CPU:
+- RAM:
+- Disk size:
+- Disk Type (HD/SDD):
+
+**Extra information**
+<!-- This is normally the contents of a `debug.log` file. Raw text or a link to a pastebin type site are preferred. -->
```

Comment:
"is" preferred, not "are" preferred. When two singular subjects are connected by "or", the verb is singular.